react/src/pages/dashboard/DatasetTypes.tsx

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
-import DatasetType from './DatasetType';
+import { makeStyles, Chip, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+    chip: {
+        margin: theme.spacing(.5)
+    },
+}));
 
 interface DatasetTypesProps {
     datasetTypes: {[key: string]: number},
 }
 
 export default function DatasetTypes({ datasetTypes }: DatasetTypesProps) {
+    const classes = useStyles();
+
+    const getChip = (type: string, number: number) => {
+        return <Chip className={classes.chip} label={type} avatar={<Avatar>{number}</Avatar>} />
+    } 
```

Comment:
Same thing here, although this is only used in one place so it might not warrant the separation at all.